### PR TITLE
remove chkconfig for el7

### LINKED ISSUE #1341: Chckconfig error on el7
While using chkconfig --list on el7 the following error message appear

Note: This output shows SysV services only and does not include native
     systemd services. SysV configuration data might be overridden by native
     systemd configuration.

```
 If you want to list systemd services use 'systemctl list-unit-files'.
 To see services enabled on particular target use
 'systemctl list-dependencies [target]'.
```

### PR DESCRIPTION
**Pull request by whitx**
# Description

Remove chkconfig for el7 as it show error message during the installation/update. (post)
# Impacts

Installation/update
# Issue
#1341
# Delete branch after merge

YES
## Bug Fixes
#1341

### CODE REVIEW

File: addons/packages/packetfence.spec [hunk 1]
```diff
@@ -653,12 +653,22 @@ fi
 
 for service in snortd httpd snmptrapd portreserve redis
 do
+  %if 0%{?el6}
```
Comment (fdurand) [inline addons/packages/packetfence.spec]: no space

File: addons/packages/packetfence.spec [hunk 2]
```diff
@@ -653,12 +653,22 @@ fi
...
   if /sbin/chkconfig --list | grep $service > /dev/null 2>&1; then
     echo "Disabling $service startup script"
     /sbin/chkconfig --del $service > /dev/null 2>&1
   fi
+  %endif
```
Comment (fdurand) [inline addons/packages/packetfence.spec]: no space

File: addons/packages/packetfence.spec [hunk 3]
```diff
@@ -653,12 +653,22 @@ fi
...
+  %if 0%{?el7}
+  if /bin/systemctl -a | grep $service > /dev/null 2>&1; then
+    echo "Disabling $service startup script"
+    /bin/systemctl disable $service > /dev/null 2>&1
+  fi
+  %endif
```
Comment (fdurand) [inline addons/packages/packetfence.spec]: no space

**Comment by julsemaan** [comment]
btw for next time put `fixes #1341` in your description and not just the issue number so it closes the bug for you

**Pull request merged**